astroquery/gaia/core.py

```diff
@@ -569,7 +627,7 @@ def __cone_search(self, coordinate, radius, *, table_name=None,
             when the job is executed in asynchronous mode, this flag specifies
             whether the execution will wait until results are available
         output_file : str, optional, default None
-            file name where the results are saved if dump_to_file is True.
+            file name where the results are saved if `dump_to_file` is True.
```

Comment:
sphinx tries to resolve the single backticks into links, and there is no API docs anchor for them, so all these needs to be in double backticks to make the docs build pass. I'm sorry about that, I'll add fixes as a commit, so this won't hold up a PR.

```suggestion
            file name where the results are saved if ``dump_to_file`` is True.
```